README.md

```diff
@@ -69,6 +69,8 @@ ros2 launch domain_bridge domain_bridge.launch.xml config:=examples/example_brid
 
 You can also override domain IDs with optional launch arguments `from_domain` and `to_domain`.
 
+To enable bi-directional bridging of a topic, set the `reverse` argument to `true`.
```

Comment:
should the argument be called `biridectional`?
That sounds more clear to me.

Reply:
Renamed to `bidirectional` in bc3461a